lib/retirement_schemes/classification_count.rb

```diff
@@ -1,11 +1,21 @@
 module RetirementSchemes
   class ClassificationCount
-    def initialize(count)
-      @count = count
+    def initialize(options = {})
+      options = options.with_indifferent_access
+
+      @count = options.fetch(:count)
+    end
+
+    def retire?(sw_count)
+      if disabled?
+        false
+      else
+        sw_count.classifications_count >= @count
+      end
     end
 
-    def retire?(count)
-      count.classifications_count >= @count
+    def disabled?
```

Comment:
is this used anymore or are we just going to overwrite the criteria scheme to NeverRetire?